lib/DAV/Browser/Plugin.php

```
@@ -157,6 +157,9 @@ public function httpGet(RequestInterface $request, ResponseInterface $response)
     public function httpPOST(RequestInterface $request, ResponseInterface $response)
     {
         $contentType = $request->getHeader('Content-Type');
+        if (!\is_string($contentType)) {
+            return;
```

Maybe throw a meaningful exception instead?

---

The code below at line 166 already just returns from this method. That is where things like an empty string for `$contentType` is currently "handled". Having `null` content type or some other thing seems just the same, so I "handled" it the same here. This case just needs to be handled separately before the `explode` because `explode` does not cope with "exploding" `null`

---

I see. Thx for elaborating.

seems like this is a bug which would have bern caught with a proper static analysis setup.. we should consider leveling up phpstan and use a baseline